client/fidelity/compliance-review/controllers/ComplianceReviewController.js

```diff
@@ -1,7 +1,8 @@
 import {get} from 'lodash'
 import {getDateFilters} from 'apps/search/directives/DateFilters'
```

Comment:
Please update this to the full path as discussed yesterday so we don't introduce a regression when releasing

Reply:
:pray: